### DISCUSSION TITLE
Strimzi operator is deleting images

### DISCUSSION THREAD

**Discussion by mariuszkowalczuk-develop** [strimzi/strimzi-kafka-operator#6673] [Q&A] [answered]
Hello, 
We are using strimzi/operator:0.20.0 and we've observed some strange behaviour.
Operator is deleting other images from the repository in order to free some space on the disk. 
![logs](https://user-images.githubusercontent.com/62005657/162925810-8dcccafa-00a5-4c00-a92c-1cf6de908e8c.png)
Is there a reason why it's behaving like this? Is there a way to overcome this?

**Comment by scholzj** [comment, marked as answer]
The operator cannot do it - it does not control it, it doesn't even pull images, it does not interact with the container engine in any way. This is something your platform takes care about. Strimzi just tells Kubernetes that it should deploy a Pod with container running image _X_. And that is it. The rest is on the platform.

**Reply by mariuszkowalczuk-develop** [reply]
Thank You for the answer.
